internals/utils/resourcePoolHelper.go

```diff
@@ -33,15 +33,22 @@ func GetNamespaceResourcePooled(namespace *ObjectContext) string {
 	if IsRootNamespace(namespace.Object) {
 		return "false"
 	}
-	namespaceSns, err := GetNamespaceSns(namespace)
-	if err != nil {
-		return "false"
-	}
-	currentState := namespaceSns.Object.(*danav1.Subnamespace).Labels[danav1.ResourcePool]
-	if currentState == "" {
+	currentNamespaceState := namespace.Object.(*corev1.Namespace).Labels[danav1.ResourcePool]
+	if currentNamespaceState == "true" {
+		return "true"
+	} else if currentNamespaceState == "false" {
 		return "false"
+	} else {
+		namespaceSns, err := GetNamespaceSns(namespace)
+		if err != nil {
+			return "false"
+		}
+		currentSnsState := namespaceSns.Object.(*danav1.Subnamespace).Labels[danav1.ResourcePool]
+		if currentSnsState == "" {
+			return "false"
+		}
+		return currentSnsState
```

Comment:
I _think_ we can just check if `currentNamespaceState` is not "true" or "false" already and then if it's not ("" for example) then we do 

```
currentSnsState := namespaceSns.Object.(*danav1.Subnamespace).Labels[danav1.ResourcePool]
```

And the just return `currentSnsState`